bitnami/moodle/README.md

```diff
@@ -52,140 +52,141 @@ The following table lists the configurable parameters of the Moodle chart and th
-| `mariadb.master.persistence.hostPath`      | Host mount path for MariaDB volume       | `nil` (will not mount to a host path)          |
-| `externalDatabase.user`                    | Existing username in the external db     | `bn_moodle`                                    |
-| `externalDatabase.password`                | Password for the above username          | `nil`                                          |
-| `externalDatabase.database`                | Name of the existing database            | `bitnami_moodle`                               |
-| `externalDatabase.host`                    | Host of the existing database            | `nil`                                          |
-| `externalDatabase.port`                    | Port of the existing database            | `3306`                                         |
+| Parameter                                   | Description                                                                                                           | Default                                                      |
+|---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| `mariadb.enabled`                           | Whether to use the MariaDB chart                                                                                      | `true`                                                       |
+| `mariadb.architecture`                      | MariaDB architecture (`standalone` or `replication`)                                                                  | `standalone`                                                 |
+| `mariadb.auth.rootPassword`                 | Password for the MariaDB `root` user                                                                                  | _random 10 character alphanumeric string_                    |
+| `mariadb.auth.database`                     | Database name to create                                                                                               | `bitnami_moodle`                                             |
+| `mariadb.auth.username`                     | Database user to create                                                                                               | `bn_moodle`                                                  |
+| `mariadb.auth.password`                     | Password for the database                                                                                             | _random 10 character long alphanumeric string_               |
+| `mariadb.replication.enabled`               | MariaDB replication enabled                                                                                           | `false`                                                      |
```

Comment:
I think this doesn't exist in the new version, right?

Reply:
Thats true, removed.

Reply:
👍